libs/rudder-integration-braze-react-native/android/common_task.gradle

```diff
@@ -59,7 +59,6 @@ repositories {
       }
     }
     google()
-    maven { url "https://appboy.github.io/appboy-android-sdk/sdk" }
```

Comment:
Are they publishing to maven central now ?

Reply:
Yes. Refer to this doc: https://github.com/braze-inc/braze-android-sdk/blob/master/CHANGELOG.md#2500